### PR TITLE
fix: semantics of LinkedList

### PR DESCRIPTION
**Pull request by delehef**

### CODE REVIEW

**Review by powerslider — APPROVED** [review]

**Review by powerslider — APPROVED** [review]

File: arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java [hunk 1]
```diff
@@ -51,19 +51,23 @@ public String toString() {
...
+    this.lists.addLast(new CountedList<>());
   }
 
   public void enter(int initialCapacity) {
-    this.lists.push(new CountedList<>(initialCapacity));
+    this.lists.addLast(new CountedList<>(initialCapacity));
+  }
+
+  public E getLast() {
+    return this.lists.getLast().getLast();
```
Comment (OlivierBBB) [inline arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java:62]: What is the explanation for the double `getLast` ?

Comment (OlivierBBB) [reply arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java:62]: BTW would `getLatest` be more descriptive and accurate than `getLast` ?

**Review by OlivierBBB — COMMENTED** [review]
I don't fully get it but if Tsvetan agrees I'll take it

File: arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java (hunk 1 above)
Comment (delehef) [reply arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/list/StackedList.java:62]: `getLast` is the default Java method name for such accessors.

**Pull request merged**